sms_sponsorship/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Quentin Gigon <gigon.quentin@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+{
+    'name': 'Compassion SMS Sponsorships',
+    'version': '10.0.1.0.3',
+    'category': 'Other',
+    'author': 'Compassion CH',
+    'license': 'AGPL-3',
+    'website': 'http://www.compassion.ch',
+    'depends': ['crm_compassion', 'website-cms'],
```

Comment:
ẁebsite-cms` is the name of the repository but is not an installable module. It should be `cms_form`. Add as well `website-cms` in the oca_dependencies.txt file